# EthernetPacketList Message

std_msgs/Header header

openpilot/EthernetPacket[] ethernetData
